python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -70,7 +70,7 @@ def test_read_json_lines(lines):
 
 
 def test_read_json_nested_experimental(tmp_path):
```

Comment:
Can we rename the tests, too? The test names referencing `experimental` should be updated.
```suggestion
def test_read_json_nested(tmp_path):
```